### DISCUSSION TITLE
The style of the user page's header looks a bit strange

### DISCUSSION THREAD

**Discussion by dsrkafuu** [community/community#22215] [Accessibility]
<p>The tabs are not aligned with the id of the avatar due to the control button’s large size:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/3/d3453df942ff4f79c76bca33bb6d7e794b46539c.png" data-download-href="/uploads/short-url/u8Z5ZTw7kpijhCayy1ebYZmcaYc.png?dl=1" title="20201209213712" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/21136293/181069579-6f636257-964f-4c89-a620-4a15419f0a1b.png" alt="20201209213712" data-base62-sha1="u8Z5ZTw7kpijhCayy1ebYZmcaYc" width="690" height="42" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3453df942ff4f79c76bca33bb6d7e794b46539c_2_690x42.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3453df942ff4f79c76bca33bb6d7e794b46539c_2_1035x63.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3453df942ff4f79c76bca33bb6d7e794b46539c_2_1380x84.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3453df942ff4f79c76bca33bb6d7e794b46539c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">20201209213712</span><span class="informations">1578×97 13.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Previous:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/6/5693bb032e0cb876e186cc53a08c0217b8e52577.png" data-download-href="/uploads/short-url/clTzz2BdcL5rSlzXgAMETc7rc1N.png?dl=1" title="20201209213726" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/21136293/181069580-18c8e36d-cdf1-4537-ace5-8f4d6b0cf183.png" alt="20201209213726" data-base62-sha1="clTzz2BdcL5rSlzXgAMETc7rc1N" width="690" height="32" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/6/5693bb032e0cb876e186cc53a08c0217b8e52577_2_690x32.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/6/5693bb032e0cb876e186cc53a08c0217b8e52577_2_1035x48.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/6/5693bb032e0cb876e186cc53a08c0217b8e52577_2_1380x64.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/6/5693bb032e0cb876e186cc53a08c0217b8e52577_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">20201209213726</span><span class="informations">1572×74 11.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks for sharing this feedback! I’ll share it with the team to review</p>

**Comment by cehfisher** [comment]
Hi @dsrkafuu - thank you for providing this feedback! After reviewing the original feedback, it seems as though issue has been resolved. Could you please confirm and close this issue? (cc @queenofcorgis)
